Vdc flex (#443)
terraform-providers/terraform-provider-vcd: CHANGELOG.md

FEATURES:

* **New Resource:** `vcd_vm_internal_disk` VM internal disk configuration

> * `vcd_vapp_vm` Internal disks in VM template allows to be edited by `override_template_disk` field

Reviewer: ```suggestion
* `vcd_vapp_vm` Internal disks in VM template can be edited by `override_template_disk` field
```

Author: changed

> * `vcd_org_vdc` allows to create Flex allocation in vCD 9.7 and later. Also new two fields added for Flex - `elasticity`, `include_vm_memory_overhead` [#443]

Reviewer 2: ```suggestion
* `vcd_org_vdc` allows to create Flex allocation in vCD 9.7 and later. Also two new fields added for Flex - `elasticity`, `include_vm_memory_overhead` [#443]
```

Author: fixed

> * `vcd_org_vdc` allows to create Flex allocation in vCD 9.7 and later. Also two new fields added for Flex - `elasticity`, `include_vm_memory_overhead` [#443]

Reviewer: ```suggestion
* `vcd_org_vdc` can be created with Flex allocation in vCD 9.7 and later. Also two new fields added for Flex - `elasticity`, `include_vm_memory_overhead` [#443]
```

Author: changed

* **New Resource:** `vcd_vm_internal_disk` VM internal disk configuration [GH-412]
* `vcd_vapp_vm` Internal disks in VM template allows to be edited by `override_template_disk` field
  [GH-412]

> * `vcd_org_vdc` can be created with Flex allocation in vCD 9.7 and later. Also two new fields added for Flex - `elasticity`, `include_vm_memory_overhead` [#443]

Reviewer 2: ```suggestion
* `vcd_org_vdc` can be created with Flex allocation in vCD 9.7 and later. Also two new fields added for Flex - `elasticity`, `include_vm_memory_overhead` [GH-443]
```

Author: fixed
